### DISCUSSION TITLE
CRC MacOS v2.8.0 can't pull image from Red Hat Registry and Docker

### DISCUSSION THREAD

**Discussion by kairu97** [crc-org/crc#3372] [Q&A]
Hi. My CRC on MacOS can't pull images from Red Hat Registry and Docker Hub. On RHEL, it works; but I don't know why on MacOS, it doesn't work. The only workaround I did is to push the image to the internal image registry of OpenShift and refer the deployment to the image in the internal registry. Is this the only alternative to do it?

**Comment by praveenkumar** [comment]
Do you also see error like https://github.com/code-ready/crc/issues/3377 one, if not can you use the issue tracker to file the issue and we can try to take a look.
